deps/build.jl

```diff
@@ -79,10 +79,9 @@ init_deps()
 end
 
 if is_apple()
-    if Pkg.installed("Homebrew") === nothing
-        error("Homebrew package not installed, please run Pkg.add(\"Homebrew\")")
-    end
     using Homebrew
+    Homebrew.update()
+    Homebrew.add("imagemagick")
```

Comment:
Where does delegate come from? Best option would be to write a bindeps validation hook function to check for something in the C library to distinguish between versions that work and versions that don't.